### PR TITLE
CIR-289 added the subject to restrictions page.

### PR DESCRIPTION
**Pull request by funkmaster20th**

### CODE REVIEW

File: test/views/GroupSubjectToRestrictionsViewSpec.scala [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+          await(nunjucksRenderer.render(GroupSubjectToRestrictionsTemplate, Json.toJsObject(YesNoRadioViewModel(form, NormalMode)))(fakeRequest))
+        } else {
+          val view = viewFor[GroupSubjectToRestrictionsView](Some(emptyUserAnswers))
+          view.apply(form, NormalMode)(fakeRequest, messages, frontendAppConfig)
+        }
+
+      behave like normalPage(applyView(form), messageKeyPrefix)
+
+      behave like pageWithBackLink(applyView(form))
+
```
Comment (peckoverHMRC) [inline test/views/GroupSubjectToRestrictionsViewSpec.scala]: needs test for subheading and save and continue link

File: conf/views/groupSubjectToRestrictions.njk [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  {{ pageTitle(form, "groupSubjectToRestrictions.title") }}
+{% endblock %}
+
+{% block content %}
+
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-two-thirds">
+
+        {% call formHelper(form, routes.controllers.GroupSubjectToRestrictionsController.onSubmit(mode).url) %}
+
```
Comment (peckoverHMRC) [inline conf/views/groupSubjectToRestrictions.njk]: subheading is missing

File: conf/app.routes [hunk 3]
```diff
@@ -121,4 +121,8 @@ POST       /full-or-abbreviated-return/change           controllers.FullOrAbbrev
...
 POST       /revising-return                             controllers.RevisingReturnController.onSubmit(mode: Mode = NormalMode)
 GET        /revising-return/change                      controllers.RevisingReturnController.onPageLoad(mode: Mode = CheckMode)
-POST       /revising-return/change                      controllers.RevisingReturnController.onSubmit(mode: Mode = CheckMode)
\ No newline at end of file
+POST       /revising-return/change                      controllers.RevisingReturnController.onSubmit(mode: Mode = CheckMode)
+GET        /groupSubjectToRestrictions                        controllers.GroupSubjectToRestrictionsController.onPageLoad(mode: Mode = NormalMode)
+POST       /groupSubjectToRestrictions                        controllers.GroupSubjectToRestrictionsController.onSubmit(mode: Mode = NormalMode)
+GET        /changeGroupSubjectToRestrictions                  controllers.GroupSubjectToRestrictionsController.onPageLoad(mode: Mode = CheckMode)
+POST       /changeGroupSubjectToRestrictions                  controllers.GroupSubjectToRestrictionsController.onSubmit(mode: Mode = CheckMode)
```
Comment (peckoverHMRC) [inline conf/app.routes]: should be kebab case

File: conf/messages.en [hunk 4]
```diff
@@ -182,3 +179,11 @@ revisingReturn.title = Are you revising a return you have already submitted?
...
 revisingReturn.checkYourAnswersLabel = Are you revising a return you have already submitted?
 revisingReturn.error.required = Select yes if you revising a return you have already submitted
+
+
+# GroupSubjectToRestrictionsPage Messages
+# ----------------------------------------------------------
+groupSubjectToRestrictions.title = Is the group subject to restrictions?
+groupSubjectToRestrictions.heading = Is the group subject to restrictions?
+groupSubjectToRestrictions.checkYourAnswersLabel = Is the group subject to restrictions?
+groupSubjectToRestrictions.error.required = Select yes if Is the group subject to restrictions?
```
Comment (yangwong9664) [inline conf/messages.en]: @funkmaster20th I believe this is incorrect, "Select yes if Is the group subject to restrictions?" does not read well. Please fix, at your earliest convenience. Many thanks.

Comment (funkmaster20th) [reply conf/messages.en]: not doing anything with content until content has been baselined

**Pull request merged**